pyscriptjs/examples/altair.html

```diff
@@ -3,8 +3,8 @@
     <title>Altair</title>
     <meta charset="utf-8">
 
-    <link rel="stylesheet" href="../build/pyscript.css" />
-    <script defer src="../build/pyscript.js"></script>
+    <link rel="stylesheet" href="build/pyscript.css" />
+    <script defer src="build/pyscript.js"></script>
```

Comment:
this should get translated to `./build/pyscript.js` by the browser. Do you mind just using `./build/pyscript.js` to make it explicitly?